setup.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+
+version = '0.0.1'
+
+setup(
+    name='multiqc_alfa',
+    version=version,
```

Comment:
why not `0.0.1` here?

Reply:
version over here is a variable, so it is `0.0.1`

Reply:
Alright, changed it to a string